1.Data-exploration/L1000/L1000_moa_median_scores_calculation.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Consensus Signatures\n",
+    "\n",
+    "A consensus signature can be defined as a perturbation-specific summary profile acquired by aggregating replicate level information.\n",
+    "\n",
+    "\n",
+    "#### Level 5 - Replicate-consensus signatures (MODZ)\n",
+    "L1000 experiments are typically done in 3 or more biological replicates. We derive a consensus replicate signature by applying the\n",
```

Comment:
I knew that I recognized this text! 

When something like this is copy and pasted, we _must_ add a citation to the source. In this case add to the markdown:

From [Subramanian et al 2017](https://doi.org/10.1016/j.cell.2017.10.049):

> L1000 experiments are typically done in 3 or more biological replicates. We derive a consensus replicate signature by applying the moderated z-score (MODZ) procedure as follows. First, a pairwise Spearman correlation matrix is computed between the replicate signatures in the space of landmark genes with trivial self-correlations being ignored (set to 0). Then, weights for each replicate are computed as the sum of its correlations to the other replicates, normalized such that all weights sum to 1. Finally, the consensus signature is given by the linear combination of the replicate signatures with the coefficients set to the weights. This procedure serves to mitigate the effects of uncorrelated or outlier replicates, and can be thought of as a ‘de-noised’ representation of the given experiment’s transcriptional consequences.

Make sure to include a URL to the DOI in the paper reference